api/tests/unit/features/test_unit_features_features_service.py

```diff
@@ -261,3 +261,38 @@ def test_feature_get_edge_overrides_data(
         overrides_data[distinct_segment_featurestate.feature.id].num_segment_overrides
         == 1
     )
+
+
+@pytest.mark.django_db(transaction=True)
+def test_get_edge_overrides_data_skips_deleted_features(
+    feature: Feature,
+    environment: "Environment",
+    identity: Identity,
+    identity_featurestate: FeatureState,
+    distinct_identity_featurestate: FeatureState,
+    dynamodb_identity_wrapper: "DynamoIdentityWrapper",
+    dynamodb_wrapper_v2: "DynamoEnvironmentV2Wrapper",
+):
+    # Given
+    # replicate identity to Edge
+    edge_identity = EdgeIdentity(map_identity_to_engine(identity, with_overrides=False))
+    edge_identity.add_feature_override(
+        map_feature_state_to_engine(identity_featurestate),
+    )
+    edge_identity.add_feature_override(
+        map_feature_state_to_engine(distinct_identity_featurestate),
+    )
```

Comment:
Perhaps we can discuss this on the standup but I'm a little unsure on the details here. Are these overrides for 2 different features? 

It's not obvious to me from the test why we're still left with one edge identity override.

Reply:
Yeah, that's for a different feature

Reply:
I think some comments would probably help here. 

```python
# Let's add 2 overrides for 2 separate features for the edge identity
edge_identity.add_feature_override(
    ...

# and delete one of the features
feature.delete()
```